variables.tf

```diff
@@ -55,26 +55,44 @@ variable session_affinity {
 
 variable ports {
   description = "List of ports range to forward to backend services. Max is 5."
-  type        = "list"
+  type        = list
 }
 
-variable http_health_check {
-  description = "Set to true if health check is type HTTP, otherwise health check is TCP."
-  default     = false
+variable "health_check" {
+  type = object({
+    type                = string
+    check_interval_sec  = number
+    healthy_threshold   = number
+    timeout_sec         = number
+    unhealthy_threshold = number
+    response            = string
+    proxy_header        = string
+    health_port         = number
+  })
+  default = null
 }
-
-variable health_port {
-  description = "Port to perform health checks on."
+variable "http_health_check" {
```

Comment:
This variable needs to be removed; it is redundant since `health_check` has `type`.